commands/files.rb

```diff
@@ -5,32 +5,40 @@
 
 class Command
   def self.files(pkg)
-    # Check if the package is even installed first, as this is the most likely reason we cannot find a filelist.
-    unless PackageUtils.installed?(pkg.name)
-      puts "Package #{pkg.name} is not installed.".lightred
-      return
-    end
-
```

Comment:
Update the help entry for the files command to reflect that this now works for non-installed packages?